src/structures/VoiceState.js

```diff
@@ -7,10 +7,6 @@ const { Error, TypeError } = require('../errors');
  * Represents the voice state for a Guild Member.
  */
 class VoiceState extends Base {
-  /**
-   * @param {Guild} guild The guild the voice state is part of
-   * @param {APIVoiceState} data The data for the voice state
```

Comment:
This was the last instance of `APIVoiceState`. Should we remove this too?

https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/VoiceState.js#L229-L232